src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/TeleporterSystem.java

```diff
@@ -41,16 +45,13 @@
  */
 @RegisterSystem(RegisterMode.AUTHORITY)
 public class TeleporterSystem extends BaseComponentSystem {
-
-    private static final Logger logger = LoggerFactory.getLogger(TeleporterSystem.class);
-
-    @In
-    InventoryManager inventoryManager;
     @In
     EntityManager entityManager;
     @In
     GameEntitySystem gameEntitySystem;
 
+    private boolean gameStart;
```

Comment:
What does this indicate?

Reply:
And is this value ever reset to `false` at the end of a game/when restarting?

Reply:
This is just a check to make sure that the game hasn't started yet, if a player joins once the game starts the pregame timer would appear on their screen in case this check isn't there. This is only when the players teleport to the base, I haven't considered the pregame phase in case of restarts yet, but this check would only be needed if the restarted game doesn't have the same amount of players.